### DISCUSSION TITLE
Admin page

### DISCUSSION THREAD

**Discussion by nickkostov** [antonym/netboot.xyz#707] [Q&A] [answered]
Hello everybody,
In terms of security, I noticed that the docker container having the netboot.xyz has no admin page.
Is there a way in which this could be implemented. When you think about it, its going to be very useful for the sake of security implementation over it. Someone could just snap their finger and mess around your with your settings and this could end up in lots of work afterwards.

If anyone has implemented such thing, could advise on the matter in order to push something like that on the local network, I am using docker-compose.

Thanks in advance for the future answers!

**Comment by antonym** [comment]
Adding @thelamer to see if he has any input as he wrote the interface.

**Comment by thelamer** [comment, marked as answer]
@nickkostov The app acts as a web based text editor essentially with some functionality to localize remote assets from Github.
The assumption is that the user would be providing their own secure gateways in the form of a reverse proxy or put it on a secure subnet. 
Authentication adds a maintenance layer to the project, and evolves into people wanting to integrate it with their existing directory solutions. 
Not functionality I am against, just not something I need or want to maintain.

**Reply by nickkostov** [reply]
Got it mate, thanks for the reply.
